src/category_theory/abelian/exact.lean

```diff
@@ -343,6 +348,91 @@ begin
   rw [fork.ι_of_ι, cofork.π_of_π, ← L.map_comp, kernel_comp_cokernel _ _ e1, L.map_zero]
 end
 
+end
+
+section
+
+variables (h : ∀ {X Y Z : A} {f : X ⟶ Y} {g : Y ⟶ Z}, exact f g → exact (L.map f) (L.map g))
```

Comment:
Have you considered using `⦃⦄` brackets here?

Reply:
Ah, thanks, forgot that those exist :grin: